turbinia/turbiniactl.py

```diff
@@ -107,11 +101,6 @@ def main():
   parser.add_argument(
       '-D', '--dump_json', action='store_true',
       help='Dump JSON output of Turbinia Request instead of sending it')
-  parser.add_argument(
```

Comment:
I think we'll still want to have a command line flag to be able to pass this in even if we move this to a recipe (similar to `--yara_rules_file` and `yara_rules` recipe item).